api/v1alpha1/routing_types.go

```diff
@@ -32,6 +32,9 @@ type RoutingSpec struct {
 	//+kubebuilder:validation:Required
 	Hostname string `json:"hostname"`
 
+	//+kubebuilder:validation:Optional
```

Comment:
```suggestion
	//+kubebuilder:validation:Optional
	//+kubebuilder:default:=true
```
and remove GetIsInternal function

Reply:
If this is true by default, it will show as true when using external addresses. The function is to avoid ambiguous config for external addresses.